### ISSUE TITLE
Login dialog does not have keyboard focus after tray icon doubleclick

### ISSUE THREAD

**Issue by xmedeko** [OpenVPN/openvpn-gui#655]
I start `openvpn-gui.exe` when on Windows start up. It does not connect automatically, just the icon is in the system tray. When I double click the icon in the tray, OpenVPN log window and then OpenVPN login dialogs popup up. The cursor is blinking in the password field of the OpenVPN login dialog, but when I try to type my password, nothing happens. I have to manually click the mouse in the password field, then I can type password and hit <kbd>Enter</kbd> to confirm the dialog.

I would like to have focus in the password field when the login dialog pops up, so I can type password immediately.

**Comment by selvanair** [comment]
I cannot reproduce this -- I get keyboard focus on the password box. What is the version of the GUI and OS?

> I start openvpn-gui.exe when on Windows start up. It does not connect automatically, just the icon is in the system tray.

In case you want the connection to auto-start, select "Enable auto restart of active connections" in the settings menu (need release 2.6.0 or later OR GUI version 11.34+). Then any connection that remains *active* when the GUI is terminated will auto-start the next time.

**Comment by xmedeko** [comment]
* Windows 11 Pro
* OpenVPN v2.6.6
* OpenVPN GUI v11.44.0.0
* Lang: cz

Yes, I have "Enable auto restart of active connections" checked. The use case is shut down computer and restart. The connection has `auth-user-pass`, do I have to type the password. That's OK. IMO the problem is, that the login dialog itself has not focus. (The password inside the dialog has focus, it has a cursor and the cursor is blinking.) It's enough to click on the login dialog top chrome, and I can type the password.

![obrazek](https://github.com/OpenVPN/openvpn-gui/assets/1101801/83092dc0-9811-473b-b05a-5a8326a9242d)

**Comment by xmedeko** [comment]
> Does this also happen if you right click on the tray icon and select connect instead of double click?

No, when I right click on the tray icon and select connect the dialog has focus, I can type the password. When I double click by left mouse button, the dialog has no focus.

**Comment by selvanair** [comment]
@xmedeko Is this still an issue? I could not reproduce it.

**Comment by xmedeko** [comment]
@selvanair You are right, I have OpenVPN GUI 11.48.0.0 and it works OK now. i.e. even double click on the icon in the system tray brings focus to the password field. (Or may it be fixed by some Windows 11 update?)
